smoker/src/smoker/backends/dummybackend.go

```diff
@@ -247,49 +251,47 @@ func NewDummyCredentialManager() CredentialManager {
 	defaultCreds := NewCleanDummyCredential()
 	creds[defaultCreds.GetUsername()] = defaultCreds
 	return &DummyCredentialManager{
-		creds:   creds,
-		current: nil}
+		Creds:   creds,
+		Current: nil}
 }
 
 // ** DummyBackend
+// *** Def
 type DummyBackend struct {
 	// Map of component ID to component
 	// Use this for most lookups
-	idLookup map[uint]Component
-	// Set of all components (for searches)
-	components map[Component]bool
+	IdLookup map[uint]Component
 	// List of all the bins
-	bins []DummyBin
+	Bins map[string]DummyBin
```

Comment:
Now Bins are randomly ordered since it's a map. I think this is a feature though, since it leads to items being randomly distributed by default, rather than put in order. Pretty easy to sort if we wish though.